skipruntime-ts/metapackage/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@skipruntime/skip",
+  "version": "0.0.1",
+  "dependencies": {
+    "@skipruntime/core": "0.0.6",
+    "@skipruntime/server": "0.0.9",
+    "@skipruntime/helpers": "0.0.9",
+    "@skipruntime/wasm": "0.0.7"
+  },
+  "optionalDependencies": {
+    "@skipruntime/native": "0.0.5",
+    "@skip-adapter/postgres": "0.0.5"
```

Comment:
Maybe just me, but wouldn't it be easier to keep track of if we made all these version numbers the same, e.g. 0.0.10? There is no issue with skipping some version numbers, right?

Reply:
Sure, but the reason we ended up in this state (and may want to continue this way) is that we sometimes update a "leaf" package without updating all of its parents.  We could enforce that everything's at the same version number, but it would mean that every release has to update everything. Not sure which way is preferable.

Reply:
I'm not sure, but I would have defaulted to something that amounts to a function from the git commit hash of the whole repo to a single version number to be used for each of the packages in the repo, perhaps just as a matter of being conservative wrt what the dependencies actually are and which combinations of package versions we test versus what users can possibly have.

I have also been trying to keep the docs site in sync with the released packages, and that would certainly be easier if each git commit corresponded to a single version number.

Reply:
Yeah, bumping everything all together whenever we make a change seems reasonable to me -- esp. with docs, if/when we start having to host multiple versioned instances of the docs site.  And it's not too much trickier to update all of the packages thento update just one package, especially if we maintain the invariant that they all share a package number.

Let's chat with rest of the team and decouple from this PR though, I think